pypdf/_writer.py

```diff
@@ -934,8 +988,14 @@ def update_page_form_field_values(
         Args:
             page: Page reference from PDF writer where the
                 annotations and field data will be updated.
-            fields: a Python dictionary of field names (/T) and text
-                values (/V)
+            fields: a Python dictionary of :
```

Comment:
Maybe the type of the `fields` parameter on line 978 should reflect those options?